[18.0][MIG] : Migration purchase_partner_incoterm to 18.0.

LucasTran380381 opened the pull request:
No change code

MiquelRForgeFlow committed:
[ADD] purchase_open_qty

MiquelRForgeFlow committed:
[10.0][MIG] purchase_open_qty: Migration to 10.0

MiquelRForgeFlow committed:
[FIX] adapt methods to some cases

MiquelRForgeFlow committed:
[ADD] More test

mourad-ehm committed:
[ADD] purchade_exception to 11.0

JordiBForgeFlow committed:
improve how is the exeption message displayed in the PO

oca-transbot committed:
OCA Transbot updated translations from Transifex

LoisRForgeFlow committed:
[10.0][IMP] purchase_open_qty: add filter to po lines

Commit:
[MIG] purchase_open_qty: Migration to 11.0

MiquelRForgeFlow committed:
[IMP] Adapt readme to smaller pieces

sudhir-erpharbor committed:
[MIG] Migrated purchase_exception to v12.

sudhir-erpharbor committed:
[IMP] Removed unused import and changed the website.

sudhir-erpharbor committed:
[IMP] Improved the code as per travis and review.

bjeficent committed:
[MIG] purchase_open_qty: Migration to v12

sudhir-erpharbor committed:
[ADD] Added Readme.

sudhir-erpharbor committed:
[IMP] Improved code.

bealdav committed:
FIX purchase_exception: update with major evolution of base_exception

sudhir-erpharbor committed:
[IMP] Improved code of test case.

sudhir-erpharbor committed:
[IMP] Improved Code.

sudhir-erpharbor committed:
Used self instead of deprecated variables.

florian-dacosta committed:
Fix apple and orange mixing error between po line and po union

JordiBForgeFlow committed:
[purchase_open_qty] introduces boolean fields to make it easier to find
orders with pending (or not pending) quantity to receive/invoice.

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-12.0/purchase-workflow-12.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-12-0/purchase-workflow-12-0-purchase_open_qty/

MiquelRForgeFlow committed:
[ADD] migration script to v12

liweijie0812 committed:
Added translation using Weblate (Chinese (Simplified))

liweijie0812 committed:
Added translation using Weblate (Chinese (Simplified))

liweijie0812 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (7 of 7 strings)

Translation: purchase-workflow-12.0/purchase-workflow-12.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-12-0/purchase-workflow-12-0-purchase_open_qty/zh_CN/

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (7 of 7 strings)

Translation: purchase-workflow-12.0/purchase-workflow-12.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-12-0/purchase-workflow-12-0-purchase_open_qty/it/

OCA-git-bot committed:
purchase_exception 12.0.1.1.0

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (7 of 7 strings)

Translation: purchase-workflow-12.0/purchase-workflow-12.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-12-0/purchase-workflow-12-0-purchase_open_qty/pt_BR/

kittiu committed:
[13.0][MIG] purchase_open_qty, black

kittiu committed:
[13.0][MIG] purchase_open_qty

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-13.0/purchase-workflow-13.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_open_qty/

OCA-git-bot committed:
pre-commit update

dong-z committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (7 of 7 strings)

Translation: purchase-workflow-13.0/purchase-workflow-13.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_open_qty/zh_CN/

eantones committed:
[FIX] purchase_open_qty: not able to filter by pending/receiving quantity at line level

Wrong filter fields at line level, the module is trying to use the
computed fields of the order instead of the line ones.

OCA-git-bot committed:
purchase_open_qty 13.0.1.0.1

didierdonze committed:
Translated using Weblate (French)

Currently translated at 100.0% (7 of 7 strings)

Translation: purchase-workflow-13.0/purchase-workflow-13.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_open_qty/fr/

didierdonze committed:
Translated using Weblate (German)

Currently translated at 100.0% (7 of 7 strings)

Translation: purchase-workflow-13.0/purchase-workflow-13.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-13-0/purchase-workflow-13-0-purchase_open_qty/de/

MiquelRForgeFlow committed:
[FIX] Delete empty " " spaces in same string line

teewux committed:
[IMP] purchase_exception: black, isort, prettier

teewux committed:
[MIG] purchase_exception: Migration to 13.0

SurachetS committed:
[MIG] purchase_open_qty: Migration to 14.0

Mat-moran committed:
Translated using Weblate (Spanish)

Currently translated at 85.7% (6 of 7 strings)

Translation: purchase-workflow-14.0/purchase-workflow-14.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-14-0/purchase-workflow-14-0-purchase_open_qty/es/

Yvesldff committed:
Translated using Weblate (French)

Currently translated at 57.1% (4 of 7 strings)

Translation: purchase-workflow-14.0/purchase-workflow-14.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-14-0/purchase-workflow-14-0-purchase_open_qty/fr_BE/

kittiu committed:
[14.0][IMP] purchase_open_qty, field optional='hide'

OCA-git-bot committed:
purchase_open_qty 14.0.1.0.1

kittiu committed:
[14.0][MIG] purchase_exceptoin

victoralmau committed:
[ADD] purchase_order_qty_change_no_recompute: Prevent recompute if only quantity has changed in sale order line

TDu committed:
[ADD] purchase_partner_incoterm

yankinmax committed:
[IMP] purchase_exception: black, isort, prettier

yankinmax committed:
[MIG] purchase_exception: Migration to 15.0

bealdav committed:
[IMP] purch_part_incoterm: add incoterm_address_id

Commit:
[MIG] purchase_open_qty: Migration to 15.0

cesar-tecnativa committed:
[MIG] purchase_order_qty_change_no_recompute: Migration to 15.0

[UPD] Update purchase_order_qty_change_no_recompute.pot

[UPD] README.rst

hparfr committed:
purchase_partner_incoterm: use commercial_partner_id

Incoterms should be on the supplier, not on the contact.

leemannd committed:
[MIG] purchase_partner_incoterm: Migration to 15.0

victoralmau committed:
[FIX] Change SavepointCase to TransactionCase

purchase_order_qty_change_no_recompute 15.0.1.0.1

CarlosRoca13 committed:
[IMP] purchase_open_qty: Quantity returned without credit note is pending to bill

If 10 units received, all units invoiced and returning 5 units, but this units are not
invoiced, the purchase has -5 units to invoice so this purchase order should appear as
pending to invoice.

CarlosRoca13 committed:
[FIX] purchase_open_qty: Quantity to invoice less than 0 has to be to invoice in purchase.order.lines too

OCA-git-bot committed:
purchase_open_qty 15.0.1.0.1

Commit:
[IMP] purchase_open_qty: remove qty_to_invoice field and computation

OCA-git-bot committed:
purchase_open_qty 15.0.2.0.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-15.0/purchase-workflow-15.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-15-0/purchase-workflow-15-0-purchase_open_qty/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (7 of 7 strings)

Translation: purchase-workflow-15.0/purchase-workflow-15.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-15-0/purchase-workflow-15-0-purchase_open_qty/it/

LoisRForgeFlow committed:
[IMP] purchase_open_qty: consider services

Qty to receive for services cannot be based on stock moves.

OCA-git-bot committed:
purchase_open_qty 15.0.2.0.1

aliciagaarzo committed:
[MIG] purchase_exception: Migration to 16.0

aliciagaarzo committed:
Add spanish translation

aliciagaarzo committed:
No allow ignoring exceptions with the "is_blocking" field checked, modify tests and check new functionality

dsolanki-initos committed:
[IMP] purchase_partner_incoterm: pre-commit stuff

dsolanki-initos committed:
[MIG] purchase_partner_incoterm: Migration to 16.0

santostelmo committed:
[MIG] purchase_open_qty: Migration to 16.0

[MIG] purchase_open_qty: Migration to 16.0

aliciagaarzo committed:
[MIG] purchase_order_qty_change_no_recompute: Migration to 16.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-16.0/purchase-workflow-16.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-16-0/purchase-workflow-16-0-purchase_open_qty/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Translated using Weblate (Spanish)

Currently translated at 100.0% (1 of 1 strings)

Translation: purchase-workflow-15.0/purchase-workflow-15.0-purchase_order_qty_change_no_recompute
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-15-0/purchase-workflow-15-0-purchase_order_qty_change_no_recompute/es/

[UPD] README.rst

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 74.1% (23 of 31 strings)

Translation: purchase-workflow-16.0/purchase-workflow-16.0-purchase_exception
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-16-0/purchase-workflow-16-0-purchase_exception/es/

josep-tecnativa committed:
[IMP] purchase_order_qty_change_no_recompute: test performance improvement
- Include context keys for avoiding mail operations overhead.

[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: purchase-workflow-16.0/purchase-workflow-16.0-purchase_partner_incoterm
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-16-0/purchase-workflow-16-0-purchase_partner_incoterm/it/

Commit:
[IMP] purchase_open_qty: pre-commit auto fixes

Commit:
[MIG] purchase_open_qty: Migrate to version 17.0

Commit:
[IMP] purchase_exception: pre-commit auto fixes

Commit:
[MIG] purchase_exception: Migrate to version 17.0

Commit:
[UPD] Update purchase_open_qty.pot

manishkumarbohra committed:
[IMP] purchase_partner_incoterm: pre-commit auto fixes

manishkumarbohra committed:
[MIG] purchase_partner_incoterm: Migration to 17.0

victoralmau committed:
[MIG] purchase_order_qty_change_no_recompute: Continue migration

- Improve documentation
- Improve tests
- Put it as rebel module

TT46595

Commit:
[UPD] Update purchase_order_qty_change_no_recompute.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

Commit:
[UPD] Update purchase_exception.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-17.0/purchase-workflow-17.0-purchase_exception
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-17-0/purchase-workflow-17-0-purchase_exception/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (30 of 30 strings)

Translation: purchase-workflow-17.0/purchase-workflow-17.0-purchase_exception
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-17-0/purchase-workflow-17-0-purchase_exception/it/

sergiocorato committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (7 of 7 strings)

Translation: purchase-workflow-17.0/purchase-workflow-17.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-17-0/purchase-workflow-17-0-purchase_open_qty/it/

ThiagoMForgeFlow committed:
[IMP] purchase_open_qty: add purchase_order_line_menu dependency

maciej-wichowski committed:
[IMP] purchase_order_qty_change_no_recompute: pre-commit auto fixes

maciej-wichowski committed:
[MIG] purchase_order_qty_change_no_recompute: Migration to 17.0

Commit:
[UPD] Update purchase_order_qty_change_no_recompute.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-17.0/purchase-workflow-17.0-purchase_order_qty_change_no_recompute
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-17-0/purchase-workflow-17-0-purchase_order_qty_change_no_recompute/

Commit:
[IMP] purchase_partner_incoterm: pre-commit auto fixes

Commit:
[MIG] Migrate purchase_partner_incoterm to 18.0

xaviedoanhduy commented:
> For being easily searchable and avoiding to duplicate efforts, please name the pull request following this pattern: [18.0][MIG] <module>: Migration to 18.0.

xaviedoanhduy reviewed (approved):
LGTM

you can delete the description of an unnecessary PR if it doesn't provide any information.
thanks

StefanRijnhart committed:
[MIG] purchase_open_qty: Migration to 18.0

Move models and views into model specific files.

yankinmax committed:
[IMP] purchase_exception: pre-commit auto fixes

yankinmax committed:
[MIG] purchase_exception: Migration to 18.0

yankinmax committed:
[IMP] purchase_exception: use BaseCommon as base class to reduce overhead from tracking

antonioburic committed:
[MIG] purchase_order_qty_change_no_recompute: Migration to 18.0

OCA-git-bot committed:
Merge PR #2524 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update purchase_order_qty_change_no_recompute.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2506 into 18.0

Signed-off-by simahawk

Commit:
[UPD] Update purchase_exception.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-18.0/purchase-workflow-18.0-purchase_exception
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-18-0/purchase-workflow-18-0-purchase_exception/

OCA-git-bot committed:
Merge PR #2431 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update purchase_open_qty.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: purchase-workflow-18.0/purchase-workflow-18.0-purchase_open_qty
Translate-URL: https://translation.odoo-community.org/projects/purchase-workflow-18-0/purchase-workflow-18-0-purchase_open_qty/

OCA-git-bot committed:
[IMP] update dotfiles
